### PR TITLE
Make batchposter use the unsafe endpoints

### PR DESCRIPTION
**Pull request by fontanka16**

### CODE REVIEW

File: generated_schemas/BatchPosterSchema.json [hunk 1]
```diff
@@ -29,6 +29,12 @@
             "title": "Rerunfailedrecords",
             "default": true,
             "type": "boolean"
+        },
+        "useSafeInventoryEndpoints": {
+            "title": "Usesafeinventoryendpoints",
+            "description": "Toggles the use of the safe/unsafe Inventory storage endpoints. Unsafe circumvents the Optimistic locking in FOLIO. Defaults to False (using the 'unsafe')",
+            "default": false,
```
[the remark this answers is not in the document]
Comment (fontanka16) [reply generated_schemas/BatchPosterSchema.json:36]: Sorry, no. Thank you.

File: generated_schemas/HoldingsMarcTransformerSchema.json [hunk 2]
```diff
@@ -61,7 +53,8 @@
         },
         "createSourceRecords": {
             "title": "Createsourcerecords",
-            "default": false,
+            "description": "Controls wheter or not to retain the MARC records in SRS.",
+            "default": true,
```
[the remark this answers is not in the document]
Comment (fontanka16) [reply generated_schemas/HoldingsMarcTransformerSchema.json:57]: I try to convince libraries not to keep them, am I think it should be something to opt in to. But on the other hand, not having the records in folio make the decision a little bit harde to make. 

What do you think?

[the remark this answers is not in the document]
Comment (bltravis) [reply generated_schemas/HoldingsMarcTransformerSchema.json:57]: I don't have a strong opinion either way. I just wanted to double check.

File: generated_schemas/BatchPosterSchema.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (fontanka16) [reply generated_schemas/BatchPosterSchema.json:36]: @bltravis the code was right, but the generated schemas were not. Fixing that now

**Review by bltravis — APPROVED** [review]

**Pull request merged**